apps/coscrad-frontend/src/app/routes/routes.ts

```diff
@@ -18,4 +18,5 @@ export const routes = {
                 `${resourcesBaseRoute}/${getResourceTypeLabelForRoutes(resourceType)}`
             ),
     },
+    Live: 'listenLive',
```

Comment:
Should `Live: ` be capitalized?

Reply:
Good catch- fixed. But now we pull it from the config.